first-deliverable/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"encoding/csv"
```

Comment:
Here a good practice, is to separe the imports with a new line and by categories, in this case go-chi packages would go at the end
```
import (
                "core packages"
 
                "your packages"
 
                "third party packages"
)
```

Reply:
I know. Tried to use it. Not sure used in all files